core/src/main/java/org/elasticsearch/rest/action/cat/RestIndicesAction.java

```diff
@@ -123,6 +127,19 @@ public RestResponse buildResponse(IndicesStatsResponse indicesStatsResponse) thr
         });
     }
 
+    private static final Set<String> RESPONSE_PARAMS;
+
+    static {
+        final Set<String> responseParams = new HashSet<>(Arrays.asList("local", "health"));
```

Comment:
This is like the one place where I miss Guava. It was really nice and fluent to make these.